tests/urlpatterns/converters.py

```diff
@@ -11,6 +11,26 @@ def to_url(self, value):
         return base64.b64encode(value).decode("ascii")
 
 
+class FakeBase64Converter:
+    regex = r"[a-zA-Z0-9+/]*={1}"
+
+    def to_python(self, value):
+        return base64.b64decode(value)
+
+    def to_url(self, value):
+        return base64.b64encode(value).decode("ascii")
+
+
+class InvalidOverrideIntConverter:
+    regex = r"[0-9]+"
+
+    def to_python(self, value):
+        return int(value)
+
+    def to_url(self, value):
+        return f"{value}"
+
+
```

Comment:
The tests don’t need extra converters, they can instead register the same converters under the existing names. This will avoid any test pollution issues from not resetting the converters (`FakeBase64Converter` being slightly different could affect another test, only revealed with a different order from `--reverse` or `--shuffle`).

I have made suggestions to this effect on the test file below.

```suggestion
```